MovieApp/Components/Common/SearchBarView.swift

```diff
@@ -79,19 +84,23 @@ class SearchBarView : UIView, UITextFieldDelegate {
     }
     
     func changeCancelButtonVisibility(visible: Bool) {
+        let animationSpeed = 0.2
         if visible {
-            cancelButton.isHidden = false
-            searchBox.snp.remakeConstraints{ make in
-                make.left.top.bottom.equalToSuperview()
-                make.right.equalTo(cancelButton.snp.left).offset(-10)
-                make.height.equalTo(40)
+            UIView.animate(withDuration: animationSpeed) { [self] in
```

Comment:
Even though it is not necessary to capture self weakly it would be recommended to do it to not create memory leaks in [specific cases](
https://stackoverflow.com/a/39251222)